src/Sylius/Behat/Context/Api/Shop/ProductReviewContext.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Shop;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductReviewContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    public function __construct(
+        ApiClientInterface $client,
+        ResponseCheckerInterface $responseChecker
+    ) {
+        $this->client = $client;
+        $this->responseChecker = $responseChecker;
+    }
+
+    /**
+     * @When I add it
+     */
+    public function iAddIt(): void
+    {
+        $this->client->create();
+    }
+
+    /**
+     * @When I want to review product :product
+     */
+    public function iWantToReviewProduct(ProductInterface $product): void
+    {
+        $this->client->buildCreateRequest();
+        $this->client->addRequestData('productCode', $product->getCode());
```

Comment:
In my previous review, I wrote that we need to keep a product code on command instead of iri, but we still should pass to the endpoint an iri as the resource identifier for API

Reply:
Solution for it: https://github.com/Sylius/Sylius/pull/12487